docs-source/content/userguide/experimental/istio.md

```diff
@@ -6,7 +6,7 @@ weight: 1
 
 #### Overview
 
-WebLogic Server Kubernetes Operator version 2.3 and later includes experimental support for Istio 1.2.2 and later.
+WebLogic Server Kubernetes Operator version 2.3 and later includes support for Istio 1.2.2 and later.
```

Comment:
Let's bump the Istio version up to the one that we will test, which I believe is 1.4.6. It's the same version as is supported by OLCNE 1.1.